WcaOnRails/app/jobs/compute_auxiliary_data.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class ComputeAuxiliaryData < TimedApplicationJob
+  queue_as :default
+
+  def perform
+    # Note: During the results posting process some results may be missing their corresponding WCA ID.
+    #       If we detect that (means someone else is posting results at the moment), we defer the computation.
```

Comment:
++ this is a solution for now, we should definitely do better when results submission process is Railsified.